components/create_comment/create_comment.jsx

```diff
@@ -418,15 +419,27 @@ export default class CreateComment extends React.PureComponent {
         const forceFocus = (Date.now() - this.lastBlurAt < fasterThanHumanWillClick);
         this.focusTextbox(forceFocus);
 
+        const serverError = this.state.serverError;
+        let ignoreSlash = false;
+        if (isErrorInvalidSlashCommand(serverError) && draft.message === serverError.submittedMessage) {
+            ignoreSlash = true;
+        }
+
+        const options = {ignoreSlash};
```

Comment:
I'm thinking this be may be better as
```
const options = {};

if (condition) {
    options.ignoreSlash = true;
}
```

Reply:
@mickmister your call. i am 0/5 on this but if we do change i feel we should be initialising with a value for ingnoreSlash as well for better readability.

```
const options = {
   ignoreSlash = false,
};

if (condition) {
    options.ignoreSlash = true;
}

Reply:
Looking at the three options in my editor, the original seems to be the cleanest looking. So I'm going to leave it how it is.